cipher_suite.go

```diff
@@ -216,7 +216,7 @@ var HashBLAKE2b HashFunc = hashFn{blake2bNew, "BLAKE2b"}
 func blake2sNew() hash.Hash {
 	h, err := blake2s.New256(nil)
 	if err != nil {
-		panic(err)
+		return nil
```

Comment:
Let's keep everything else but revert these `return nil`s as I think they will either panic 100% of the time or never.

Reply:
Will do. I'll update the PR shortly.